README.md

```diff
@@ -738,10 +790,29 @@ PARTITION INFO:
     Print the device's partition table.
 
 SYNOPSIS:
-    picotool partition info -m <family_id> [device-selection]
+    picotool partition info [-m <family_id>] [device-selection]
 
 OPTIONS:
-        -m <family_id> [device-selection]
+        -m <family_id>
+            family ID (will show target partition for said family)
```

Comment:
"partition" -> "partition(s)", as I guess there might be multiple partitions with the same family-ID? :shrug:

Reply:
No, it will just give you the target partition for that family ID, as if you'd dragged and dropped a UF2 with that ID. It'll do all the A/B stuff etc, and just return one partition